thunder.install

```diff
@@ -951,3 +951,138 @@ function thunder_update_8120() {
   // Output logged messages to related channel of update execution.
   return $thunderUpdater->logger()->output();
 }
+
+/**
+ * Switch to paragraphs experimental widget.
+ */
+function thunder_update_8121() {
+  /** @var \Drupal\thunder_updater\Updater $thunderUpdater */
+  $thunderUpdater = \Drupal::service('thunder_updater');
+
+  /** @var \Drupal\Core\Extension\ModuleHandlerInterface $moduleHandler */
+  $moduleHandler = \Drupal::moduleHandler();
+
+  /** @var \Drupal\Core\Extension\ModuleInstallerInterface $moduleInstaller */
+  $moduleInstaller = \Drupal::service('module_installer');
+
+  // Few modules are required only for importing paragraphs type icons and they
+  // can be uninstalled afterwards.
+  $uninstall_after_content_import = [];
+  if ($moduleHandler->moduleExists('default_content')) {
+    $uninstall_after_content_import[] = 'default_content';
+  }
+  if ($moduleHandler->moduleExists('better_normalizers')) {
+    $uninstall_after_content_import[] = 'better_normalizers';
+  }
+
+  // Update is successful only when all steps are successful.
+  $successful = TRUE;
+
+  // There are few new modules required for experimental paragraphs widget.
+  if (
+    !$thunderUpdater->installModules(['thunder__thunder_update_8121' => 'default_content'])
+    || !$thunderUpdater->installModules(['thunder__thunder_update_8121' => 'better_normalizers'])
+    || !$thunderUpdater->installModules(['thunder__thunder_update_8121' => 'paragraphs_features'])
+  ) {
+    $successful = FALSE;
```

Comment:
This check leads to the problem, that if either default_content or better_normalizers is missing, the whole update fails. Since we would still be able to do most of the updates, this should be changed.
Proposal: 
Split update hook into multiple update hooks, each of them with different dependencies that define if the update was successful. This way we can do most updates, that for example only need the paragraphs feature modules, and let only the update fail, that actually needs default_content or better_normalizers. Also, the Liveblog update could be separated out and only done when needed.
The general idea is to separate those updates that have independent requirements.